src/Sylius/Behat/Context/Ui/Admin/ManagingCountriesContext.php

```diff
@@ -69,20 +98,99 @@ public function iAddIt()
     }
 
     /**
-     * @Then I should be notified about success
+     * @When I enable it
+     */
+    public function iEnableIt()
+    {
+        $this->countryUpdatePage->enable();
+    }
+
+    /**
+     * @When I disable it
+     */
+    public function iDisableIt()
+    {
+        $this->countryUpdatePage->disable();
+    }
+
+    /**
+     * @When I save my changes
+     */
+    public function iSaveMyChanges()
+    {
+        $this->countryUpdatePage->saveChanges();
+    }
+
+    /**
+     * @Then I should be notified about successful creation
+     */
+    public function iShouldBeNotifiedAboutSuccessfulCreation()
+    {
+        $doesSuccessMessageAppear = $this->notificationAccessor->hasSuccessMessage();
```

Comment:
Please remove this variable, it does not add much readability, it is used only once and naming it properly seems like a challenge. Agree?